scout/server/blueprints/api/views.py

```diff
@@ -17,19 +19,54 @@ def case(institute_id, case_name):
     return Response(json_util.dumps(case_obj), mimetype="application/json")
 
 
+def _lookup_variant(
+    institute_id: str, case_name: str, variant_id: str
```

Comment:
```suggestion
    institute_id: str, case_name: Optional[str], variant_id: Optional[str]
```

Reply:
Hm, almost; it's the variant that is mandatory. I reorder here as well.